### PR TITLE
Add more APIv2 tests for images.

### PR DESCRIPTION
**Pull request by Edward5hen**
Covering:
push
tag
untag
rmi
image tree

Signed-off-by: Edward Shen <weshen@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
I think gitvalidation is complaining that your commit subject includes trailing whitespace

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: test/apiv2/12-imagesMore.at [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+t POST "libpod/images/nonesuch/tag?repo=myrepo&tag=mytag" '' 404
+
+# Tag the image
+t POST "libpod/images/$IMAGE/tag?repo=localhost:5000/myrepo&tag=mytag" '' 201
+
+t GET libpod/images/$IMAGE/json 200 \
+  .RepoTags[1]=localhost:5000/myrepo:mytag
+
+# Run registry container
+podman run -d --name registry -p 5000:5000 docker.io/library/registry:2.6 /entrypoint.sh /etc/docker/registry/config.yml
```
Comment (edsantiago) [inline test/apiv2/12-imagesMore.at:26]: Please stop/rm this when finished

**Review by edsantiago — COMMENTED** [review]
LGTM, but one request for cleanup. Thank you for your work on this!

File: test/apiv2/12-imagesMore.at (hunk 1 above)
Comment (Edward5hen) [reply test/apiv2/12-imagesMore.at:26]: After use it will be removed in line 38 - `t DELETE libpod/containers/registry?force=true 204`

**Comment by edsantiago** [comment]
@Edward5hen just checking in to make sure you're aware of the CI failures here

**Comment by Edward5hen** [comment]
@edsantiago sorry, I'm replying late. I was taking PTO.

I'm confused about why we have this make error. It works quite well on my local machine.

**Comment by rhatdan** [comment]
@Edward5hen Could you rebase and see if the build works.

**Comment by rhatdan** [comment]
Thanks @Edward5hen 
/lgtm

**Pull request merged**